packages/webview_flutter/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -32,6 +33,15 @@ Future<void> main() async {
       request.response.writeln('${request.headers}');
     } else if (request.uri.path == '/favicon.ico') {
       request.response.statusCode = HttpStatus.notFound;
+    } else if (request.uri.path == '/http-basic-authentication') {
+      final bool isAuthenticating = request.headers['Authorization'] != null;
+      if (isAuthenticating) {
+        request.response.writeln('Authorized');
```

Comment:
We should probably test that we actually got the expected values. E.g., this test would pass even if an implementation accidentally swapped the username and password values.